content/tools/api/AboutRestAPI.md

```diff
@@ -9,22 +9,29 @@ Contrast has a few API versions:
+The API is available to both **SaaS** and **Enterprise-on-Premises** (EOP) users. The [endpoint for SaaS users](https://app.contrastsecurity.com/Contrast/api) is the same for everyone; for EOP users, it depends on where the Contrast application is installed on the client's network.
 
->**Note:** The set of APIs is subject to change at any time, uses different keys, and in general is more about writing data to your Contrast account.
+We're always adding more endpoints, but if you'd like to see something added to the API, [file a ticket](https://support.contrastsecurity.com/tickets/new) and let's talk about it! 
+
+>**Note:** The set of APIs is subject to change at any time and use different keys to write data to your Contrast account.
 
 ## Open API Documentation
-All of Contrast's public API endpoints can be viewed [online](https://api.contrastsecurity.com/). We’ve provided open API documentation that provides resources in understanding how to use the endpoints and help you along with authenticating, navigating, collecting data and taking actions on TeamServer programmatically. This guidance is built continuously and stays current with our latest SaaS release. The same documentation can also be found embedded in the Contrast UI for both SaaS and EOP customers. Simply authenticate with an administrator account in Contrast, select **Settings**, then select **API**. 
 
-## Using the Documentation
-Endpoints and the related objects used by the endpoints are listed under “API” and “Objects”. “Flow” provides a list of sample use cases and the information of their sequence of endpoints that need be called to accomplish them.
+Contrast provides open API documentation that includes resources for understanding how to use the endpoints and help with authenticating, navigating, collecting data and taking actions on Contrast programmatically. This guidance is built continuously and stays current with our latest SaaS release. You can view all of Contrast's public API endpoints online at the [Contrast API page](https://api.contrastsecurity.com/) and the Contrast interface.
+
+### Use the Documentation
+
+Go to the **API** and **Objects** sections to see a list of endpoints and related objects used by the endpoints. The **Flow** section provides a list of sample use cases and information on their sequence of endpoints that must be called to accomplish them. The documentation also includes a playground to test your endpoint interactions with v1, v2 and v3 API endpoints. This is a handy tool to quickly learn how to use the endpoints through experience. 
+
+To access the API documentation in the Contrast interface, authenticate with an administrator account in Contrast, go to the ** User menu > Organization Settings > API tab**, and click the button to **Launch Documentation**. Unlike the static Contrast API page, this version allows you to use actual data from your Contrast application to exercise the documentation. 
```

Comment:
@jday3 Can you verify wording here?

Reply:
Should be "this version executes API requests against a real server."